install/changelog.md

```diff
@@ -112,6 +112,22 @@ While a `tsconfig.json` file is no longer required, as Pulumi uses intelligent d
 - The CLI no longer emits warnings if it can't detect metadata about your git enlistement (for example, what GitHub project it coresponds to).
 - The CLI now only warns about adding a plaintext configuration in cases where it appears likely you may be storing a secret.
 
+### Pulumi Service
+
+Since the last release, the Pulumi Service made some key improvements around the handling of update logs. When viewing stack update logs that include the output of other
```

Comment:
No need for "Since the last release" - that's implicit in this being a changelog.

Reply:
Done.